etc/help.txt

```diff
@@ -195,15 +195,26 @@ Tcl commands (Available in GUI or Batch console or Batch script):
                               : Uses alternate bitstream generation configuration files
 </openfpga>
 
----------------
---- Program ---
----------------
+------------------
+--- Programmer ---
+------------------
    programmer <command> 
```

Comment:
Please add a short description for "programmer" tcl command like others.
e.g. _add_litex_ip_catalog <directory> : Browses directory for LiteX IP generators, adds the IP(s) to the IP Catalog_

Reply:
Added short description.